qualtran/bloqs/arithmetic.py

```diff
@@ -488,7 +488,7 @@ def signature(self) -> Signature:
         return Signature.build(x=self.bitsize, target=1)
 
     def t_complexity(self) -> TComplexity:
-        return t_complexity(LessThanGate(self.bitsize, val=self.val))
+        return t_complexity(LessThanGate(self.bitsize, less_than_val=self.val))
```

Comment:
how was this not caught? can we add a test?

Reply:
done.